Content.Server/Medical/DefibrillatorSystem.cs

```diff
@@ -59,6 +57,10 @@ private void OnAfterInteract(EntityUid uid, DefibrillatorComponent component, Af
     {
         if (args.Handled || args.Target is not { } target)
             return;
+
+        if (!TryComp(uid, out UseDelayComponent? useDelay) || _useDelay.IsDelayed((uid, useDelay)) || _useDelay.IsDelayed((uid, useDelay), component.DelayId))
```

Comment:
why is it checked 2 delays? shouldnt it just be checking the specific one for the zapping? also, should this not be in CanZap?

Reply:
It was checking the other delay because I thought it would need so you couldn't use it when it is turning on. After testing though, that appears not to be the case?

And yeah it should probably be in CanZap!